templates/core/terraform/resource_processor/vmss_porter/cloud-config.yaml

```diff
@@ -39,12 +39,18 @@ write_files:
       AZURE_TENANT_ID=${arm_tenant_id}
       ARM_USE_MSI=true
       APPLICATIONINSIGHTS_CONNECTION_STRING=${app_insights_connection_string}
-  # a weekly cron job to have docker free disk space
-  - path: /etc/cron.weekly/docker-prune
+  - path: /etc/cron.hourly/docker-prune
+    # An hourly cron job to have docker free disk space. Running this frquently
+    # since disk might get full fast, but we prune only when free space is low.
     content: |
       #!/bin/bash
       set -o errexit
-      docker system prune -f
+      used_percent=$(df / --output=pcent | tail -1 | sed 's/[^0-9]//g')
+      echo "Used disk space percent: $${used_percent}"
+      if (( used_percent > 60 )); then
```

Comment:
60 seems a bit low, why not 80 or so?

Reply:
The machine comes online with a volume of 29GB, 20% free.
80% means less than 6GB free.
Since we have images 1.5 GB or even 3+ (Guacamoles VMs) I thought it will be better to have 60% set for pruning.
If we continue to see space errors we'll increase the disk since (and the threshold too).